### PR TITLE
corearoadbike.com

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [X] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

 - https://github.com/AdguardTeam/AdguardFilters/issues/138560

### Add your comment and screenshots

 - https://www.similarweb.com/website/corearoadbike.com/#traffic

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -63,6 +63,9 @@ allets.ru,annahelp.ru,newslenta.com,kogdavyydet.com,u74.ru,wifika.ru,mater.com.u
...
 !
+corearoadbike.com##img[onclick*="twitter.com/share"]
+corearoadbike.com##img[onclick*="facebook.com/sharer/sharer.php"]
+corearoadbike.com##form[name="form_view"] ~ table[align="center"]
```
Comment (zloyden) [inline SocialFilter/sections/specific.txt:68]: The site does not open even with VPN. Could you provide some screenshots for those rules?

Comment (piquark6046) [reply SocialFilter/sections/specific.txt:68]: With DevTool?

Comment (zloyden) [reply SocialFilter/sections/specific.txt:68]: Yes, it would be nice.

Comment (piquark6046) [reply SocialFilter/sections/specific.txt:68]: <details><summary>Screenshots</summary>

Screenshot3
Screenshot1
Screenshot2

</details>

Do you need more screenshot?

Comment (zloyden) [reply SocialFilter/sections/specific.txt:68]: They look ok. I don't need anymore screenshots. thanks!

**Review by zloyden — APPROVED** [review]

**Pull request merged**